nextflow.config

```diff
@@ -48,7 +44,7 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes'
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```

Comment:
Was this intentional? Pretty sure we removed this trailing slash not that long ago..

Reply:
Yes that was intentional, added this while bug hunting - copied it from the nf-core/rnaseq config.
But it didn't change anything of course so I'll remove again 👍 
Thanks for the review!

Reply:
Rnaseq needs fixing then. It breaks something, can't remember what now..